### PR TITLE
Adding an additional request to show business organization info.

### PR DESCRIPTION
**Pull request by pedrohfsd**
#### What does this PR do? \*
Shows the business organization information that was missing.

#### How to test it? \*
Login with any user that has a Manager role, on the top menu select My Account->My Users and check if the name, email, telephone and address attributes are correctly showing on screen.

### CODE REVIEW

**Comment by pedrohfsd** [comment]
> Could you add a workspace to test? Also, I think your GitHub user is not configured in your machine
> image

Oops, thanks I may have forgotten the --global keyword in git config.
I'm using the following workspace to test: https://unileverb2b--vtexspain.myvtex.com/account#/users
You'll have to login in order to test this feature, send me a message on slack so i can pass you the credentials.

File: react/components/MyUsers.tsx [hunk 1]
```diff
@@ -23,8 +23,8 @@ import {
   BUSINESS_ROLE_FIELDS,
   BUSINESS_ROLE_SCHEMA,
   ORG_ASSIGNMENT,
-  ORG_ASSIGNMENT_FIELDS,
-  ORG_ASSIGNMENT_SCHEMA,
+  // ORG_ASSIGNMENT_FIELDS,
+  // ORG_ASSIGNMENT_SCHEMA,
```
Comment (Klynger) [inline react/components/MyUsers.tsx:27]: Why not remove these lines?

File: react/components/MyUsers.tsx [hunk 2]
```diff
@@ -235,7 +225,7 @@ const MyUsers = ({
                     <UserListItem
                       isCurrentUserAdmin={isCurrentUserAdmin}
                       isDefaultAssignment={
-                        defaultUserAssignment.id == assignment.id
+                        defaultUserAssignment?.id == assignment.id
```
Comment (Klynger) [inline react/components/MyUsers.tsx:228]:
```suggestion
                        defaultUserAssignment?.id === assignment.id
```

**Comment by Klynger** [comment]
Also it needs to rebase

File: react/components/MyUsers.tsx (hunk 1 above)
Comment (pedrohfsd) [reply react/components/MyUsers.tsx:27]: Done!

File: react/components/MyUsers.tsx (hunk 2 above)
Comment (pedrohfsd) [reply react/components/MyUsers.tsx:228]: Done!

**Comment by pedrohfsd** [comment]
> Also it needs to rebase

Due to a talk that must be done with Clouda we are opting no to rebase right now release a tag for testing. Rebase will done after getting feedback from Clouda.

**Pull request merged**